.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Pull Request Template related to Kubeflow manifests Issues
+
+🔄 Please include a summary of changes and the related issue. List any dependencies that are required for this change. Please delete the options that are not relevant.
```

Comment:
Can we itemize that?

Tasks:
- Include a summary of changes and the related issue
- List any dependencies that are required for this change
- Delete the options that are not relevant.